assets/dev/js/editor/views/add-section/base.js

```diff
@@ -6,12 +6,14 @@ import environment from 'elementor-common/utils/environment';
  */
  class AddSectionBase extends Marionette.ItemView {
 	static IS_CONTAINER_ACTIVE = ! ! elementorCommon.config.experimentalFeatures.container;
+	static IS_CONTAINER_GRID_ACTIVE = ! ! elementorCommon.config.experimentalFeatures.container_grid;
 
 	// Views.
 	static VIEW_CHOOSE_ACTION = 'choose-action';
 	static VIEW_CONTAINER_FLEX_PRESET = 'select-container-preset';
 	static VIEW_CONTAINER_GRID_PRESET = 'select-container-preset-grid';
-	static VIEW_SELECT_PRESET = ( AddSectionBase.IS_CONTAINER_ACTIVE ) ? 'select-type' : 'select-preset';
+	static VIEW_SELECT_TYPE_OR_PRESET = ( AddSectionBase.IS_CONTAINER_GRID_ACTIVE ) ? 'select-type' : 'select-container-preset';
+	static VIEW_SELECT_PRESET = ( AddSectionBase.IS_CONTAINER_ACTIVE ) ? AddSectionBase.VIEW_SELECT_TYPE_OR_PRESET : 'select-preset';
```

Comment:
I find the current structure a bit difficult to read because of the 'chain reasoning'. I am thinking an normal if/else-if/else structure, might be easier to read.

Reply:
Maybe we should just do this if Hein finds it easier to read perhaps?
```
static VIEW_SELECT_TYPE_OR_PRESET = ( AddSectionBase.IS_CONTAINER_GRID_ACTIVE ) 
    ? 'select-type'
    : 'select-container-preset';
```

Reply:
maybe extract to functions
getSelectType() 
getSelectTypePreset()